engine/src/variable.cpp

```diff
@@ -1770,6 +1770,10 @@ void MCVarref::clearuql()
 		ref->clearuql();
 	else if (!isparam)
 		handler->getvar(index, isparam)->clearuql();
+    // SN-2015-02-25: [[ Bug 14536 ]] Reinstate the UQL unsetting
+    //  for the container as well.
+    else
+        handler->getcontainer(index, isparam)->getvar()->clearuql();
```

Comment:
Does handler -> getvar() not return the referenced variable for an @ parameter, if so then this change is the same as removing the (!isparam) guard.